webapp/src/components/answerValue/AnswerValue.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { withTranslation } from 'react-i18next';
+
+import MultiFileValue from './MultiFileValue'
+
+const baseUrl = process.env.REACT_APP_API_URL;
+const FILE = "file";
+const INFORMATION = 'information'
+const MULTI_CHOICE = "multi-choice";
+const MULTI_FILE = 'multi-file';
+
+function AnswerValue({answer, question, t}) {
+  if (answer && answer.question_type) {
+    if (answer.question_type === INFORMATION) {
+      return "";
+    }
+  } else {
+    if (question && question.type === INFORMATION) {
+      return "";
+    }
+  }
+
+  if (answer && answer.value) {
+    if (typeof answer.value === "string") {
+      answer.value.trim()
```

Comment:
This line does nothing since `trim()` doesn't mutate the string...

Reply:
Great catch. I missed the particular usage of it at https://github.com/deep-learning-indaba/Baobab/blob/21c48b2758f023753a332838e635db85edd98e51/webapp/src/pages/review/components/ReviewForm.js#L81.

Removed in https://github.com/deep-learning-indaba/Baobab/commit/75065206aff293b5617352429adb6ef3a1762376.